src/sql/util.py

```diff
@@ -157,6 +157,113 @@ def show_deprecation_warning():
     )
 
 
+def check_duplicate_arguments(
+    magic_execute, cmd_from, args, allowed_duplicates=[], disallowed_aliases={}
```

Comment:
`[]` and `{}` should not be used as defaults as they'll be reused across calls.

instead, set them to none and initialize them  inside the function

```
something = None

something = something or []

```

Reply:
Fixed [here](https://github.com/ploomber/jupysql/pull/908/files#diff-653c15e6161c97f356be04e072838ed15014af275c06019a7692ab727fa44b97R187-R188)